Gemfile

```diff
@@ -4,6 +4,8 @@
 
 source "https://rubygems.org"
 
+gem 'jazzy'
```

Comment:
There should be a corresponding change to `Gemfile.lock` here which specifies the exact version of `jazzy` to use. You just need to run `bundle install` and it will generate this change.

Reply:
https://github.com/ably/ably-cocoa/pull/1428/commits/ae671e231bb31486cf88756b6baf070c6d1efb7f